### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by Lawmsangi**
1. I  create a new branch according to the GitHub flow rules.
2. I set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
3. I add node_modules to .gitignore.
4. I add index.html with h1 tag containing "Hello Microverse!".
5. I give color to h1 tag using CSS.
6. I add test.md file.
7. I add test.md file to .gitignore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,164 @@
-# tests
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (Zuhaib042) [inline README.md]: - [ ] kindly consider removing this line from here as having broken images in our readme make it unprofessional. if you have your own logo, you can place it here.

File: README.md [hunk 2]
```diff
@@ -1 +1,164 @@
...
+  cd hello-microverse
+  npm install
+
+### Usage
+
+To run the project, execute the following command:
+  open index.html
+### Run tests
+
+To run tests, run the following command:
```
Comment (Zuhaib042) [inline README.md]: - [ ] Kindly consider removing this line no 99, as it has no meaning when you write in the next line that there is no test.

File: README.md [hunk 3]
```diff
@@ -1 +1,164 @@
...
+  no test
+
+### Deployment
+
+You can deploy this project using:
```
Comment (Zuhaib042) [inline README.md]: [ ] Kindly consider removing this line no 104, as it has no meaning when you write in the next line that there is no deployment.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Lawmsangi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Professional Readme

## Required Changes ♻️
- [x] Kindly consider writing descriptive commit messages in imperative tense, so instead of `Added homepage`, you can write `Add homepage to project`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Johnadibe — APPROVED** [review]
# Status: Approved ✔️
### Hi @Lawmsangi👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

LeonardoInGatsby

## Highlights
- Good commit history✔️
- Descriptive PR✔️
- Professional README Documentation✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**